plugins/inputs/x509_cert/java_key_store_test.go

```diff
@@ -0,0 +1,193 @@
+package x509_cert
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"math/big"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/pavlo-v-chernykh/keystore-go/v4"
+	"github.com/stretchr/testify/require"
+	"software.sslmate.com/src/go-pkcs12"
+
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/testutil"
+)
+
+type SelfSignedCert struct {
+	CertPEM []byte
+	KeyPEM  []byte
+	CertDER []byte
+}
```

Comment:
Please do not export this struct or the fields!
```suggestion
type selfSignedCert struct {
	certPEM []byte
	keyPEM  []byte
	certDER []byte
}
```